radionice/pocetni/lekcije/004_osobne_zamjenice_i_oslovljavanje/004_osobne_zamjenice_i_oslovljavanje.tex

```diff
@@ -123,27 +129,21 @@
 
 	\ten \fukudai{Nastavci vezani uz društevni položaj}
 
-
-
 	-さん	koristimo za nepoznate osobe i osobe prema kojima želimo biti pristojni, te općenito za odrasle osobe. Neke imenice već u sebi imaju nastavak さん i obično iskazuju poštovanje prema onom na što se odnose.
 	
-	
-	\vspace{10pt}	
-	
+	\begin{table}[!h]	
 	\begin{tabular}{|l|l|}
 		\hline
 		おじょうさん&kćer, mlada dama\\\hline
 		みなさん&svi\\\hline
 		おおやさん&stanodavac, stanodavka\\\hline
 
 	\end{tabular}
-	
-	\vspace{10pt}
+	\end{table}
 	
 	Kada pričamo o obitelji, koristimo drugačije nazive ovisno o tome da li govorimo o svojoj ili tuđoj obitelji. Kada govorimo o svojoj obitelji, govorimo o ljudima koji pripadaju našem unutarnjem krugu, a kada govorimo o tuđoj obitelji, govorimo o ljudima koji pripadaju vanjskom krugu i sukladno s time postoje drugačiji nazivi za članove obitelji.
```

Comment:
U ovom odlomčiću imam dojam da smo utrošili puno više riječi nego što je potrebno da se kaže ono što smo htjeli reći. Možda bi bilo dobro presložiti.

Reply:
Pretvoreno u: 

Ovisno o tome govorimo li o ljudima iz naše ili iz tuđe obitelji trebamo koristiti prikladne nazive za članove obitelji.